src/main/java/uk/gov/ons/ctp/response/action/service/CaseNotificationService.java

```diff
@@ -126,24 +156,31 @@ private void saveActionCase(ActionCase actionCase) {
     actionCaseRepo.save(actionCase);
   }
 
-  private void updateActionCase(UUID actionCaseId, UUID actionPlanId) {
+  private void updateActionCase(
+      UUID actionCaseId, UUID actionPlanId, CaseNotification notification) {
     ActionCase existingCase = actionCaseRepo.findById(actionCaseId);
     if (existingCase == null) {
       log.with("case_id", actionCaseId.toString()).error("No case found to update");
       throw new IllegalStateException(
           String.format(ACTION_CASE_NOT_FOUND, actionCaseId.toString()));
     }
-    ActionPlan actionPlan = actionPlanRepo.findById(actionPlanId);
-    if (actionPlan == null) {
-      log.with("action_plan_id", actionPlanId.toString()).error("No action plan found");
-      throw new IllegalStateException(
-          String.format(ACTION_PLAN_NOT_FOUND, actionPlanId.toString()));
+    if (actionPlanId != null) {
```

Comment:
Likewise, would be good to get a comment to make it easier to tear this code out after

Reply:
Sure 👍